oak_functions/examples/key_value_lookup/README.md

```diff
@@ -0,0 +1,14 @@
+# Oak Functions `key_value_lookup` example
+
+This example shows the simplest nontrivial application of Oak Functions that
+uses the key / value lookup functionality.
+
+For each incoming client request, it performs an individual lookup using the
+in-memory key / value store, and returns the item, if found, back to the client.
+If the item with the provided key is not found, it returns an empty response.
+
+Both request and response are opaque byte arrays.
```

Comment:
Might be useful to add a paragraph here about reading the lookup from stdin and perhaps even a code snippet on how to use the client. Or, alternatively a readme in the client directory to cover that.